File: src/main/java/de/projekt/priorityplanner/model/UserStory.java
package de.projekt.priorityplanner.model;

public class UserStory {
    private String name;
    private String beschreibung;
    private int value1;
    private int value2;
    private int zeit;

    public UserStory(String name, String beschreibung, int value1, int value2, int zeit) {
        this.name = name;
        this.beschreibung = beschreibung;
        this.value1 = value1;
        this.value2 = value2;
        this.zeit = zeit;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBeschreibung() {
        return beschreibung;
    }

    public void setBeschreibung(String beschreibung) {
        this.beschreibung = beschreibung;
    }

    public int getValue1() {
        return value1;
    }

    public void setValue1(int value1) {
        this.value1 = value1;
    }

    public int getValue2() {
        return value2;
    }

    public void setValue2(int value2) {
        this.value2 = value2;
    }

    public int getZeit() {
        return zeit;
    }

    public void setZeit(int zeit) {
        this.zeit = zeit;
    }
}

File: src/main/resources/static/js/UserStory.js

class UserStory{


     constructor(name, beschreibung, id) {
         this._name = name;
         this._beschreibung = beschreibung;
         this._id = id;
     }

     get id(){
         return this._id;
     }
     set id(id){
         this._id = id;
     }
     get name() {
         return this._name;
     }

     set name(value) {
         this._name = value;
     }

     get beschreibung() {
         return this._beschreibung;
     }

     set beschreibung(value) {
         this._beschreibung = value;
     }

     get value1() {
         return this._value1;
     }

     set value1(value) {
         this._value1 = value;
     }

     get value2() {
         return this._value2;
     }

     set value2(value) {
         this._value2 = value;
     }

     get zeit() {
         return this._zeit;
     }

     set zeit(value) {
         this._zeit = value;
     }

     get string(){
         let string = [this.name, this.beschreibung, this.value1, this.value2, this.zeit];
         //return string.join(", ");
         console.log(string.join(", "));
    }
 }